tests/tests/runner.rs

```diff
@@ -151,3 +168,56 @@ async fn file_data_sources() {
     let stop_block = test_ptr(2);
     ctx.start_and_sync_to(stop_block).await;
 }
+
+#[tokio::test]
+async fn template_static_filters_false_positives() {
+    let stores = stores("./integration-tests/config.simple.toml").await;
+
+    let subgraph_name = SubgraphName::new("dynamic-data-source").unwrap();
+    let hash = {
+        let test_dir = format!("./integration-tests/{}", subgraph_name);
+        fixture::build_subgraph(&test_dir).await
+    };
+
+    let blocks = {
+        let block_0 = genesis();
+        let block_1 = empty_block(block_0.ptr(), test_ptr(1));
+        let block_2 = empty_block(block_1.ptr(), test_ptr(2));
+        vec![block_0, block_1, block_2]
+    };
+    let stop_block = test_ptr(1);
+    let chain = Arc::new(chain(blocks, &stores).await);
+
+    let mut env_vars = EnvVars::default();
+    env_vars.experimental_static_filters = true;
+
+    let ctx = fixture::setup(
+        subgraph_name.clone(),
+        &hash,
+        &stores,
+        chain,
+        None,
+        Some(env_vars),
+    )
+    .await;
+    ctx.start_and_sync_to(stop_block).await;
+
+    let poi = ctx
+        .store
+        .get_proof_of_indexing(&ctx.deployment.hash, &None, test_ptr(1))
+        .await
+        .unwrap();
```

Comment:
This is testing the PoI value at block 1, but it's worth also comparing to the value at block 0 or block 2 and ensure it doesn't change.